docs/4-frameworks/03-Vue.md

```diff
@@ -2,76 +2,59 @@
 
 In this tutorial you will learn how to add UI5 Web Components to your application. You can add UI5 Web Components both to new Vue.js applications and to already existing ones.
 
-### Step 1. Install Vue CLI.
+## Setting up a Vite and Vue.js project with UI5 Web Components
+<br>
 
-```bash
-npm install -g @vue/cli
-```
-
-### Step 2. Init a new app.
-
-```bash
-vue create ui5-web-components-application
-cd ui5-web-components-application
-```
+### Step 1. Setup a Vue project with Vite.
 
-### Step 3. Add UI5 Web Components.
+To initialize a Vue project using Vite, please follow the instructions provided in the [official Vite documentation.](https://vitejs.dev/guide/#scaffolding-your-first-vite-project)
```

Comment:
Isn't more correct to reference the Vue docs 
https://vuejs.org/guide/quick-start.html
or just
https://vuejs.org

"In this section we will introduce how to scaffold a Vue [Single Page Application](https://vuejs.org/guide/extras/ways-of-using-vue.html#single-page-application-spa) on your local machine. The created project will be using a build setup based on [Vite](https://vitejs.dev/)"

Reply:
I reffered to vite documentation, since we created the section about setting up a Vite specific project, and therefore a showcase about Vite configuration for our web components is present.

Shall I edit it and create it for a default Vue config ?

Reply:
You are right, it would be more correct. I missed that. Thanks